.swiftformat

```diff
@@ -87,6 +87,7 @@
     # OPTIONAL
     # acronyms, \ - Difficult to use because it messes with coding keys etc. So a lot of disabling this rule would be necessary
     blankLineAfterSwitchCase, \
+    blankLinesBetweenImports, \
```

Comment:
I would disable this rule. I really like to have `import` and `@testable import` separated and sorted within their blocks.